android/BOINC/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Mon Dec 28 10:00:20 PST 2015
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
```

Comment:
Is not 2.14.1 a very old gradle version? Current gradle version, as of writing this, is 6.5.1.

Reply:
I am compiling old version. 7.4.53 (not 7.16.3 let's say) it very old version that this code was written before there was any android studio (and compile with  eclipse).
After my change you will able to compile old version of the code with the latest android studio (not latest gradle, I am sorry).